filebeat/config/config_test.go

```diff
@@ -82,10 +83,15 @@ func TestGetConfigFiles_Dir(t *testing.T) {
 	files, err := getConfigFiles(absPath)
 
 	assert.Nil(t, err)
-	assert.Equal(t, 2, len(files))
+	assert.Condition(t, func() bool { return len(files) >= 2 })
```

Comment:
Yes, I added `invalid.yml` file in the same directory, used by one of the system test.

I rewrote the unit tests to check for the existence of `config.yml` and `config2.yml`. But there might be more files.